addons/event/data/email_template_data.xml

```diff
@@ -169,8 +169,12 @@
     </table>
     % if object.event_id.address_id:
     <div style="width:598px;margin:0px auto;border-left:1px solid #dddddd;border-right:1px solid #dddddd;border-bottom:1px solid #dddddd;">
-        <a href="https://maps.google.com/maps?q=${location}" target="new">
-            <img src="http://maps.googleapis.com/maps/api/staticmap?autoscale=1&size=598x200&maptype=roadmap&format=png&visual_refresh=true&markers=size:mid%7Ccolor:0xa5117d%7Clabel:%7C${location}" style="vertical-align:bottom;" />
+        <a href="${object.event_id.google_map_link()}" target="new" style="text-decoration:none;color:#875A7B;">
```

Comment:
if you are going to break the translations in stable version, please update the .pot files

Reply:
> if you are going to break the translations in stable version, please update the .pot files

I have updated the .pot files, please have a look

Reply:
Thanks !